base/field_abilities/Wave.ts

```diff
@@ -27,14 +27,21 @@ export class Wave {
     }
 
     resize(width: number, height: number) {
-        this.width = width;
-        this.height = height;
+        this.width = Math.round(width);
```

Comment:
Why rounding width and height?

Reply:
For some reason, It wouldn't scale without the numbers being rounded

Reply:
Understood. Use `| 0`  for better performance then: `this.width = width | 0;`